sorc/ncep_post.fd/INITPOST_NETCDF.f

```diff
@@ -53,7 +53,8 @@ SUBROUTINE INITPOST_NETCDF(ncid2d,ncid3d)
               smstot, ivgtyp, isltyp, sfcevp, sfcexc, acsnow, acsnom, sst, thz0, qz0,           &
               uz0, vz0, ptop, htop, pbot, hbot, ptopl, pbotl, ttopl, ptopm, pbotm, ttopm,       &
               ptoph, pboth, pblcfr, ttoph, runoff, tecan, tetran, tedir, twa, maxtshltr,        &
-              mintshltr, maxrhshltr, fdnsst,            &
+              mintshltr, maxrhshltr, fdnsst, acgraup, graup_bucket, acfrain, frzrn_bucket,      &
```

Comment:
@ericaligo-NOAA Can you add a change log with your name at the start session of this subroutine? e.g. !> 2022-11-16 | Eric James    | ...